src/frontend/src/icp-eth/components/tokens/ManageTokens.svelte

```diff
@@ -150,7 +157,12 @@
 					{token.symbol}
 				</span>
 
-				<IcManageTokenToggle slot="action" {token} on:icToken={onToggle} />
+				<!-- We add to break it like this because it was giving a wrong rendering with the IF statement -->
```

Comment:
What do you mean? "We ad to break it like" what?

Reply:
it is a typo: We **had** to break it...

but basically, if i put the IF statement by itself like this:

```svelte
{#if isIcrcCustomToken(token)}
  <IcManageTokenToggle slot="actions" {token} on:icToken={onToggle} />
{/if}
```

the layout result is wrong, and apparently it is really the IF statement: i tested with and without, and that was the difference

<img width="707" alt="Screenshot 2024-06-19 at 09 27 13" src="https://github.com/dfinity/oisy-wallet/assets/169057656/892741a2-e116-40eb-82c2-7007891c1992">

Reply:
Ah then you can remove the comment. That's standard Svelte 4 behavior. You indeed need a fragment for the slot if you use such a if.

Reply:
```
if
   div slot // Not allowed
   
fragment slot
   if
      div // Allowed
```

Reply:
ah ok, i didn't know that... removing

 is there some doc on that for me to read? i swear I searched a lot but nothing explicitly saying that

Reply:
Don't know where we learned it anymore spontaneously.

It's deprecated in Svelte v5 anyway 😜